packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -468,72 +565,109 @@ body {
 }
 
 //----------------
-// Detached
+// 7. Panel Header
 //----------------
-.aa-DetachedSearchButton {
+.aa-PanelHeader {
   align-items: center;
+  background: var(--aa-primary-color);
+  color: #fff;
+  display: grid;
+  height: var(--aa-modal-header-height);
+  margin: 0;
+  padding: var(--aa-spacing-half) var(--aa-spacing);
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(
+      var(--aa-background-color),
+      var(--aa-background-color-alpha-0)
+    );
+    bottom: calc(var(--aa-spacing-half) * -1);
+    content: '';
+    height: var(--aa-spacing-half);
+    left: 0;
+    pointer-events: none;
+    position: absolute;
+    right: 0;
+    z-index: 99;
```

Comment:
How did we pick the `z-index` values? I've seen, several values, some of them one, two, or three digits.

It might be easier to manage with a [Tailwind-style scale](https://tailwindcss.com/docs/z-index).

Reply:
random 🤙

Reply:
One thing that can happen is users who have really strong `z-index` of their own and it sets some of their UI elements on top of the Autocomplete ([example in detached mode](https://codesandbox.io/s/elegant-dubinsky-9ddqh?file=/src/index.css)).

What we could do is set a `--aa-base-z-index` that the user can override with a value that's higher than any of those they have in their project, and calculate the ones that need to always be on top with `calc`.

```css
:root {
  --aa-base-z-index: 9999;
  --detached-modal-z-index: calc(var(--aa-base-z-index) * 0.5);
}
```

Reply:
Good idea. I did it this way, wdty? 9c260268e7ff881df3b879c694f98ca25e0e9f95